L1Trigger/Phase2L1ParticleFlow/plugins/DeregionizerProducer.cc

```diff
@@ -170,19 +173,5 @@ void DeregionizerProducer::setRefs_(l1t::PFCandidate &pf, const l1ct::PuppiObjEm
   }
 }
 
-void DeregionizerProducer::fillDescriptions(edm::ConfigurationDescriptions &descriptions) {
```

Comment:
Why is `fillDescriptions()` removed?

Reply:
I would hazard a guess that it was removed because of the extremely complicated `linkConfigs` configuration added into the configuration here: https://github.com/cms-sw/cmssw/pull/41279/files#diff-5361519969ac836bcfcd1766c340fc1f82ffdf021ff12943f216334ebc692870R56. I'm willing to bet that there was no simple solution to making a default value for this particular configuration, so the entirety of the description was removed.

Reply:
It would be possible to keep the rest and define the `linkConfigs` as "vector of ParameterSets that can contain anything" (combination of https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#ParameterSets_that_Contain_Vecto and https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Let_the_ParameterSetDescription)

Reply:
@cerminar to speed up the integration, could you please answer why did you remove the fillDeacriptions method here in DeregionizerProducer.cc (and possibly restore it, following perhaps the suggestion of @makortel here above)?

Reply:
The code change actually from @thesps, but authorship has become tricky after too many rebases in the 6+ months from when it was authored to when it got here. That said, Andrew's guess was most likely right.

If re-adding the method is required for this code to be integrated, I can make a PR to [aloeliger:l1t-1063](https://github.com/aloeliger/cmssw/tree/l1t-1063) to fix it.

Reply:
@gpetruc Thanks for this. If you would like me to do it, just let me know. Otherwise I will wait for your PR to my branch.

Reply:
https://github.com/aloeliger/cmssw/pull/3

Reply:
This should be present now after merging this PR